www/includes/easyparliament/member.php

```diff
@@ -502,7 +502,7 @@ public function party_text($party = null) {
         }
     }
 
-    public function entered_house($house = 0) {
+    public function entered_house($house = HOUSE_TYPE_ROYAL) {
```

Comment:
Similarly, here (and `current_member` below) should really have `$house = null` or similar, royal should not be the default, this is a legacy issue.

Reply:
same here, I'll revert this change